tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -391,13 +392,33 @@ public ContainerLaunchContext createAMContainerSpec(ApplicationId appId, long am
     acls.put(ApplicationAccessType.MODIFY_APP, " ");
     amContainer.setApplicationACLs(acls);
 
+    String command = TonyClient.buildCommand(amMemory, taskParams, pythonBinaryPath, pythonVenv,
+        executes, hdfsClasspath, shellEnv, containerEnv);
+
+    LOG.info("Completed setting up Application Master command " + command);
+    List<String> commands = new ArrayList<>();
+    commands.add(command);
+    amContainer.setCommands(commands);
+    if (tokens != null) {
+      amContainer.setTokens(tokens);
+    }
+    amContainer.setEnvironment(containerEnv);
+    amContainer.setLocalResources(localResources);
+
+    return amContainer;
+  }
+
+  @SuppressWarnings("WeakerAccess")
```

Comment:
Can we remove this and just make the method package-private (no modifier) instead?
```
static String buildCommand(...) { ... }
```